util/test/demos/vk/vk_shader_debug_zoo.cpp

```diff
@@ -2834,6 +2834,31 @@ OpBranch %_bottomlabel
 
 )EOTEST",
     });
+
+    // test switch for different integer types
+    std::vector<std::string> intTypes = {"int", "uint"};
+    std::vector<std::string> caseLiterals = {"0x12345678", "0xF2345678"};
+    if(features.shaderInt64)
+    {
+      intTypes.push_back("i64");
+      intTypes.push_back("u64");
+      caseLiterals.push_back("0x1234567812345678");
+      caseLiterals.push_back("0xF234567812345678");
+    }
+    for(size_t i = 0; i < intTypes.size(); ++i)
+      append_tests({fmt::format(
```

Comment:
Since this one-line is quite long, I'd prefer to have `{}` around the inner loop.